src/devices/rpLidar2/rpLidar2.h

```diff
@@ -61,9 +61,22 @@ class RpLidar2 : public RateThread, public yarp::dev::IRangefinder2D, public Dev
     rplidardrv*           m_drv;
 
 public:
-    RpLidar2(int period = 10) : RateThread(period)
-    {
-    }
+    RpLidar2(int period = 10) : RateThread(period),
+        m_sensorsNum(0),
+        m_buffer_life(0),
+        m_min_angle(0.0),
+        m_max_angle(0.0),
+        m_min_distance(0.0),
+        m_max_distance(0.0),
+        m_resolution(0.0),
+        m_clip_max_enable(false),
+        m_clip_min_enable(false),
+        m_do_not_clip_infinity_enable(false),
+        m_inExpressMode(false),
+        m_pwm_val(0),
+        m_device_status(DEVICE_OK_STANBY),
```

Comment:
I did not check this, but "STANBY" looks like a typo...

Reply:
That's the actual name of the status https://github.com/robotology/yarp/blob/2fae7144ff2b329bc20bf9819b965d98c7a33124/src/libYARP_dev/include/yarp/dev/IRangefinder2D.h#L38

Reply:
:+1:

Reply:
Opened issue #1469